database/migrations/2017_07_10_150510_shift_failed_jobs_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class ShiftFailedJobsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('failed_jobs', function (Blueprint $table){
```

Comment:
@mattstauffer It unfortunately can't be. This was done this way to address a really whacky SQLite issue. You can read more about it [here](https://laracasts.com/discuss/channels/general-discussion/migrations-sqlite-general-error-1-cannot-add-a-not-null-column-with-default-value-null). My implementation was taken from the last comment in that thread. Definitely a strange one!